dle/data/management/commands/load_ema_data.py

```diff
@@ -0,0 +1,266 @@
+from django.core.management.base import BaseCommand, CommandError
+from data.models import DrugLabel, LabelProduct, ProductSection
+from django.core.files.base import ContentFile
+from django.core.files.storage import default_storage
+from django.conf import settings
+import requests
+import fitz  # PyMuPDF
+from bs4 import BeautifulSoup
+import re
+import datetime
+
+# pip install bs4
+# pip install pymupdf
```

Comment:
Will we need to install these packages in our virtual environments? If so it might be a good idea to throw them in the requirements.txt file

Reply:
Yes, that's a great point. I will plan to add the dependencies to requirements.txt